views/partials/_example.njk

```diff
@@ -59,13 +59,63 @@
 	<div class="app-tabs__heading js-tabs__heading{{ " js-tabs__heading--open" if (params.open) }}"><a href="#{{ exampleId }}-nunjucks" role="tab" aria-controls="{{ exampleId }}-nunjucks" data-track="tab-nunjucks">Nunjucks</a></div>
   {% endif %}
   <div class="app-tabs__container js-tabs__container{{ " js-tabs__container--no-tabs" if (params.hideTab) }}" id="{{ exampleId }}-nunjucks" role="tabpanel">
+    {%- if (params.group == 'components') %}
+      {% set macroOptions = getMacroOptions(params.item) %}
+
+      {% set macroOptionsHTML %}
+
+        <p>
+        Use options to customise the appearance, content and behaviour of a component when using a macro, for example, changing the text.
+        </p>
+        <p>
+          Some options are required for the macro to work; these are marked as "Required" in the option description.
+        </p>
+
+        <p>
+          If you’re using Nunjucks macros in production be aware that using html options, or ones ending with Html can be a <a href="https://developer.mozilla.org/en-US/docs/Glossary/Cross-site_scripting">security risk</a>. 
+        </p>
+
+        {% for table in macroOptions %}
+
+          <table class="govuk-table app-options__table" id="nunjucks-options-table-{{table.id}}">
```

Comment:
Because this table is included multiple times on the same page, this is resulting in non-unique IDs, which is also causing issues linking to other tables when the 'default' example is not the first example in the page.

(As discussed in person)

Reply:
Thanks @36degrees for the excellent brain storming - fixed now.